### DISCUSSION TITLE
algolia-react-native-Error -View config getter callback for component `input` must be a function

### DISCUSSION THREAD

**Discussion by swachchhakh** [algolia/instantsearch#5357] [Q&A] [answered]
![Screenshot_4](https://user-images.githubusercontent.com/49757436/161113527-af8873c6-1312-434e-889b-fa7a9b96f4f4.png)

![Screenshot_1648741532](https://user-images.githubusercontent.com/49757436/161113124-01bfcf18-176f-4797-967c-444350c46a9e.png)

**Comment by dhayab** [comment, marked as answer]
Hi, quite a late answer to this, but React InstantSearch widgets are not compatible with React Native. You need to use connectors to recreate the UI. Since then, we released React InstantSearch Hooks which makes this simpler, by allowing you to leverage the Hooks we use to build our web UI components. Check out the guide on how to [use React InstantSearch Hooks with React Native](https://www.algolia.com/doc/guides/building-search-ui/going-further/native/react-hooks/) for more information.
